packages/cce/package.py

```diff
@@ -0,0 +1,13 @@
+from spack import *
+
+
+class Cce(Package):
+    """Cray compiler
```

Comment:
Put inheritance here at least, otherwise it will be really bug proned

Reply:
Not possible, there's no builtin Cray compiler package.